### ISSUE TITLE
使用 llm-export导出qwen的onnx后，example无法直接运行

### ISSUE THREAD

**Issue by BaofengZan** [inisis/OnnxLLM#1]
`import os 
import torch
import random
import numpy as np 

DEFAULT_RANDOM_SEED = 1024

def seedBasic(seed=DEFAULT_RANDOM_SEED):
    random.seed(seed)
    os.environ['PYTHONHASHSEED'] = str(seed)
    np.random.seed(seed)

def seedTorch(seed=DEFAULT_RANDOM_SEED):
    torch.manual_seed(seed)
    torch.cuda.manual_seed(seed)
    torch.backends.cudnn.deterministic = True
    torch.backends.cudnn.benchmark = False

def seedEverything(seed=DEFAULT_RANDOM_SEED):
    seedBasic(seed)
    seedTorch(seed)
    
seedEverything()

from transformers import AutoTokenizer
import sys
sys.path.insert(0, "/data/Learning/mnn-llm/OnnxLLM")
from onnxllm import AutoModelForCausalLM

tokenizer = AutoTokenizer.from_pretrained("/data/Learning/mnn-llm/llm-export-master/model/qwen/Qwen1___5-1___8B-Chat/", trust_remote_code=True)
model = AutoModelForCausalLM.from_pretrained("/data/Learning/mnn-llm/llm-export-master/onnx/", trust_remote_code=True)

prompt = '蒙古国的首都是乌兰巴托（Ulaanbaatar）\n冰岛的首都是雷克雅未克（Reykjavik）\n埃塞俄比亚的首都是'

inputs = tokenizer(prompt, return_tensors='pt')

output = model.generate(**inputs, max_new_tokens=50)
print(tokenizer.decode(output[0], skip_special_tokens=True))
`

报错：
<img width="823" alt="image" src="https://github.com/user-attachments/assets/1f570d49-31d5-420a-9d07-5b0fad1828ac">

**Comment by inisis** [comment]
Hi, currently you can not directly use model exported by llm-export, you should use https://huggingface.co/inisis-me/Qwen-1_8B instead, or you can copy the files under https://huggingface.co/inisis-me/Qwen-1_8B except for the *.onnx, it should work fine.
